File: Java/Easy/1406-subtract-the-product-and-sum-of-digits-of-an-integer/subtract-the-product-and-sum-of-digits-of-an-integer.java
class Solution {
    public int subtractProductAndSum(int n) {
        int sum=0,product=1,digit=0,diff=0;
        while(n!=0){
            digit=n%10;
            sum+=digit;
            product*=digit;
            n=n/10;
        }
        diff=product-sum;
        return diff;
    }
}

File: Python/Easy/1406-subtract-the-product-and-sum-of-digits-of-an-integer/subtract-the-product-and-sum-of-digits-of-an-integer.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
       int num=n,sum=0,product=1;
       while(num>0){
       sum+=num%10;
       product*=num%10;
       num=num/10;
    }
    return product-sum;
    }
};